frontend/src/constants.ts

```diff
@@ -9,5 +9,7 @@ export const ONCHAIN_DUST_SATS = 1000;
 export const ALBY_HIDE_HOSTED_BALANCE_BELOW = 100;
 export const ALBY_MIN_HOSTED_BALANCE_FOR_FIRST_CHANNEL = 30_000;
 
+export const LIST_TRANSACTIONS_LIMIT = 10;
```

Comment:
10 is quite small, maybe 20 or 50 is good?

Reply:
I think 20 should be good, changed!